org.eclipse.xtext.xbase/src/org/eclipse/xtext/xbase/typesystem/internal/IFeatureScopeTracker.java

```diff
@@ -37,12 +37,12 @@
 	}
 	
 	/**
-	 * @see IResolvedTypes#getExpressionScope(EObject, Anchor)
+	 * @see IResolvedTypes#getExpressionScope(EObject, IExpressionScope.Anchor)
```

Comment:
wondering why this qualification is necessary, since Anchor is imported. But if you saw an error before...

Reply:
thats why i have changed that

Reply:
Nested types cannot be referenced directly from JavaDoc as far as I know.